packages/resolver/lib/sources/truffle/Deployed.ts

```diff
@@ -46,14 +46,6 @@ export class Deployed {
       //remove "payable"s in conversions if we're before 0.6.0
       source = source.replace(/payable\((.*)\)/g, "$1");
     }
-    //regardless of version, replace all pragmas with the new version
```

Comment:
Oh! Wait! Crap! I just realized, if we're going to get rid of this, we should also change the pragma above so it supports Solidity 0.4.x.  So, um, please make that change.  We can just make it `pragma solidity >= 0.4.15 < 0.9.0`, I figure.  (It won't work before 0.4.15 because of the `pure` keyword.)

Reply:
Cool, I updated it!